tests/test_cases/test_cocotb/test_logging.py

```diff
@@ -75,3 +75,12 @@ async def test_logging_default_config(dut):
         # Restore pre-test configuration
         os.environ = os_environ_prev
         cocotb_log.level = log_level_prev
+
+
+@cocotb.test()
+async def test_custom_logging_levels(dut):
+    logging.basicConfig(level=logging.NOTSET)
+    logging.addLevelName(5, "SUPER_DEBUG")
+    logger = logging.getLogger("name")
+    logger.setLevel(5)
+    logger.log(5, "SUPER DEBUG MESSAGE!")
```

Comment:
The style checker wants a newline at the end of the file, but I'm not sure if I can suggest that with the GitHub GUI without it adding an empty line.